upstairs/src/lib.rs

```diff
@@ -1401,15 +1516,13 @@ where
             }
             DsState::OnlineRepair => {
                 drop(ds);
+                // Start some task here to handle the requests
```

Comment:
Does this need an XXX or TODO?

Reply:
TODO: Add a TODO here.

Reply:
fixed in 02d694383914e8a615080888c1b1bcf9de1be8a3